subsys/bluetooth/controller/ll_sw/nordic/lll/lll_vendor.h

```diff
@@ -5,8 +5,18 @@
  */
 
 #define EVENT_OVERHEAD_XTAL_US        1500
-#define EVENT_OVERHEAD_PREEMPT_US     0    /* if <= min, then dynamic preempt */
+#define EVENT_OVERHEAD_PREEMPT_US     0    /* if <= min, then dynamic preempt*/
 #define EVENT_OVERHEAD_PREEMPT_MIN_US 0
 #define EVENT_OVERHEAD_PREEMPT_MAX_US EVENT_OVERHEAD_XTAL_US
 #define EVENT_OVERHEAD_START_US       200
+/* Worst-case time margin needed after event end-time in the air
+ * (done/preempt race margin + power-down/chain delay)
+ */
+#define EVENT_OVERHEAD_END_US         40
 #define EVENT_JITTER_US               16
+
+#define EVENT_RX_JITTER_US(phy) 16    /* Radio Rx timing uncertainty */
+#define EVENT_RX_TO_US(phy) (((((phy)&0x03) + 4)*8 / BIT((((phy)&0x3)>>1))) + \
+				  EVENT_RX_JITTER_US(phy))
+#define EVENT_RX_TX_TURNARROUND(phy)  ((phy) == 1?100:((phy) == 2 ? 80:900))
```

Comment:
100,80,900 us: How sure are we (Oticon) about these (Nordic) timing constants?

Reply:
Not at all. I was not sure how to handle that aspect, so I chose to simply add the numbers that Oticon uses, and expected for Nordic to change these to be correct.